eslint.config

```diff
@@ -0,0 +1,13 @@
+{
+  "parserOptions": {"ecmaVersion": 6},
+  "env": {
+    "browser": true,
+  },
+  "extends": "airbnb-base",
+  rules: {
+    "no-shadow": 0,
+    "no-param-reassign": 0,
+    "eol-last": 0,
+    "arrow-parens": "as-needed"
```

Comment:
This should be `"arrow-parens": ["error", "as-needed"]` or `"arrow-parens": [2, "as-needed"]`